glean-core/src/lib.rs

```diff
@@ -190,13 +190,19 @@ impl Glean {
         let data_store = Some(Database::new(&cfg.data_path, cfg.delay_ping_lifetime_io)?);
         let event_data_store = EventDatabase::new(&cfg.data_path)?;
 
+        // Create an upload manager with rate limiting of 10 pings every 60 seconds.
+        let mut upload_manager = PingUploadManager::new(&cfg.data_path, false);
+        upload_manager.set_rate_limiter(
```

Comment:
Should we also add a rate limiter to the upload manager created in [`glean_initialize_standalone_uploader`](https://github.com/mozilla/glean/blob/50cf80311315caf05f4870397e83812a5eda5731/glean-core/ffi/src/lib.rs#L480). I have just noticed this, but I am not sure exactly from looking at the code if it is necessary. If so I can open a follow up bug for it. @badboy ?

Reply:
@brizental yes, we should. Please kindly file a follow-up :)